samples/snippets/src/main/java/com/example/bigtable/Reads.java

```diff
@@ -200,6 +201,39 @@ public static void readPrefix(String projectId, String instanceId, String tableI
   }
   // [END bigtable_reads_prefix]
 
+  // [START bigtable_reverse_scan]
+  public static void readRowsReversed() {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "my-project-id";
+    String instanceId = "my-instance-id";
+    String tableId = "mobile-time-series";
+    readPrefix(projectId, instanceId, tableId);
+  }
+
+  public static void readRowsReversed(String projectId, String instanceId, String tableId) {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
+    try (BigtableDataClient dataClient = BigtableDataClient.create(projectId, instanceId)) {
+      Query query = Query.create(tableId)
+          .reversed(true)
+          .limit(2)
+          .prefix("phone#4c410523")
+          .range(ByteStringRange.unbounded()
+              .startClosed("phone#5c10102")
```

Comment:
startClosed and end closed is the same key, maybe change to different keys? So customers can know if start key should be < end key or > end key because it's reversed?

Reply:
Or maybe in the javadoc example in Query.java update it to:

```
   * <pre>{@code
   * query
   *   .range(ByteStringRange.unbounded().startKeyClosed("a").endOpen("c"))
   *   .limit(10)
   *   .reversed(true)
   * }</pre>
```

Reply:
I updated the sample to use different keys

Reply:
(in https://github.com/googleapis/java-bigtable/pull/1711)